_data/flux-commands.yaml

```diff
@@ -119,6 +121,41 @@ commands:
             command: flux alloc -N 2 --queue=foo
           - title: Create an allocation in the background with two nodes on queue 'foo'
             command: flux alloc --bg -N 2 --queue=foo
+
+  - name: flux bulksubmit
+    groups:
+      - name: Format String Examples
+        items:
+          - title: Pipe inputs into flux bulksubmit
+            command: seq 1 4 | flux bulksubmit echo {}
+          - title: Use GNU parallel to map the same inputs in
+            command: "flux bulksubmit echo {} ::: 1 2 3 4"
+          - title: Remove the extension of the input string (e.g., one)
+            command: "flux bulksubmit echo {.%} ::: one.txt two.txt three.txt"
+          - title: Get the basename of the input string (e.g., path.txt)
+            command: "flux bulksubmit echo {./} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Combine the two - get the basename with extension removed (e.g., path)
+            command: "flux bulksubmit echo {./%} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the directory name of the input string (e.g., /long/file)
+            command: "flux bulksubmit echo {.//} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the input sequence number, start counting at 0
+            command: "flux bulksubmit echo {seq} ::: 1 2 3 4"
+          - title: Get the input sequence number, start counting at 1 (you monster)
+            command: "flux bulksubmit echo {seq1} ::: 1 2 3 4"
+          - title: Define a custom counter called p2
+            command: "flux bulksubmit --dry-run --define=p2='2**int(x)' -n {.p2} hostname \
+   ::: $(seq 0 4)"
+
+      - name: Simple Examples
+        items:
+          - title: Compress every file under ./mydir in parallel, one process per core
+            command: find ./mydir | flux bulksubmit --wait xz {} 
+          - title: Do a dry run and provide multiple inputs using GNU parallel
+            command: "flux bulksubmit --dry-run /bin/bash -c 'touch mydir/{0}; echo touched number {1}' ::: filea fileb filec filed ::: 1 2 3 4"
+          - title: Use flux 'carbon copy' to submit identical jobs with different inputs
+            command: flux submit --cc="1-10" echo "Hello I am job {cc}"
```

Comment:
Idea: perhaps instead just show the bulksubmit example and add a fyi, could also be done with flux submit --cc like this ...

Also, could add some --cc examples to flux submit, and do a similar "you could also use bulksubmit like this ..."

Reply:
I think if people are coming to the cheat sheet looking for "how do I submit my jobs in bulk" it would be ok to have the `--cc` example here!

But we definitely could add more examples up there - I only saw the one example from your docs, but if you want to do another follow up PR to add more extensive ones that would be great!

Reply:
> I think if people are coming to the cheat sheet looking for "how do I submit my jobs in bulk" it would be ok to have the --cc example here!

I see your point.  I guess `bulksubmit` is simply the name we use for the gnu parallel style input.  Internally the guts are identical to `--cc` with submit.

Reply:
Hm, I wouldn't say the internals are the same as `--cc`, in fact, with `bulksubmit` you can substitute into the `--cc` argument:
```
$ flux bulksubmit --dry-run --cc={1} echo {} ::: a b c ::: 0-1 0-3 0-7
bulksubmit: submit --cc=0-1 echo a
bulksubmit: submit --cc=0-3 echo a
bulksubmit: submit --cc=0-7 echo a
bulksubmit: submit --cc=0-1 echo b
bulksubmit: submit --cc=0-3 echo b
bulksubmit: submit --cc=0-7 echo b
bulksubmit: submit --cc=0-1 echo c
bulksubmit: submit --cc=0-3 echo c
bulksubmit: submit --cc=0-7 echo c
```

The reason we didn't call it `flux parallel` or something like that is because "parallel" is overloaded in the context of Flux, since you can use to launch tasks in parallel, where this tool submits jobs in parallel. In that case `bulksubmit` seemed more appropriate and less confusing.